resources/views/graph.blade.php

```diff
@@ -41,6 +42,50 @@
 {{--Canvas for where the chart will be displayed--}}
 <div class="container">
   <canvas id="graph"></canvas>
+   <br/><br/><br/>
+  <div class"row justify-content-center">
+    <table width="100%">
+      <tr>
+        <th width="5%"></th>
+        <th width="5%"></th>
+        <th width="5%"></th>
+        <th width="5%"></th>
+        <th width="5%"></th>
+        <th width="5%"></th>
+        <th width="5%"></th>
+        <th width="5%"></th>
+        <th width="5%"></th>
+        <th width="5%"></th>
+      </tr>
+      <tr>
+        <?php
```

Comment:
try to use blade template tag @for or convert <?php to @php if the php block is really unavoidable